minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "server_wildcards": [
+      "play.crystopia.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
```

Comment:
Not reachable

Reply:
Oh yes sorry i fix the site soon... Sorry I hope that is not bad)